src/Nethermind/Nethermind.Merge.AuRa.Test/AuRaMergeEngineModuleTests.cs

```diff
@@ -40,47 +40,37 @@ public class AuRaMergeEngineModuleTests : EngineModuleTests
-    // This happens because for this AuRa chain the blockHash includes AuRa specific fields, hence the hash for genesis is different
-    // causing all subsequent blocks to have a different blockHash.
-    // You can verify this by removing `SealEngineType = Nethermind.Core.SealEngineType.AuRa;` from the constructor of
-    // the test class above and rerunning the tests.
     [TestCaseSource(nameof(GetWithdrawalValidationValues))]
-    public override async Task newPayloadV2_should_validate_withdrawals((
-        IReleaseSpec Spec,
+    public override Task forkchoiceUpdatedV2_should_validate_withdrawals((IReleaseSpec Spec,
         string ErrorMessage,
         IEnumerable<Withdrawal>? Withdrawals,
         string BlockHash
         ) input)
-    {
-        await Task.CompletedTask;
-    }
-
-    [Test]
-    public override async Task Should_process_block_as_expected_V2()
-    {
-        await Task.CompletedTask;
-    }
-
-    [Test]
-    public override async Task processing_block_should_serialize_valid_responses()
-    {
-        await Task.CompletedTask;
-    }
+        => base.forkchoiceUpdatedV2_should_validate_withdrawals(input);
+
+    [TestCase(
+        "0xe168b70ac8a6f7d90734010030801fbb2dcce03a657155c4024b36ba8d1e3926",
+        "0x60049df788e4d9473a2547e40c3188ee464c9f8855a8d82b724f3a199d70c325",
+        "0x03e662d795ee2234c492ca4a08de03b1d7e3e0297af81a76582e16de75cdfc51",
+        "0x78ecfec08729d895")]
+    public override Task Should_process_block_as_expected_V2(string latestValidHash, string blockHash, string stateRoot, string payloadId)
+        => base.Should_process_block_as_expected_V2(latestValidHash, blockHash, stateRoot, payloadId);
+
+    [TestCase(
+        "0xe4333fcde906675e50500bf53a6c73bc51b2517509bc3cff2d24d0de9b8dd23e",
+        "0xe168b70ac8a6f7d90734010030801fbb2dcce03a657155c4024b36ba8d1e3926",
+        "0x78ecfec08729d895")]
+    public override Task processing_block_should_serialize_valid_responses(string blockHash, string latestValidHash, string payloadId)
+        => base.processing_block_should_serialize_valid_responses(blockHash, latestValidHash, payloadId);
 
     [Test]
-    public override async Task forkchoiceUpdatedV1_should_communicate_with_boost_relay_through_http()
-    {
-        // NOTE: This is the blockhash AuRa produces `0xb337e096b1540ade48f63104b653691af54bb87feb0944d7ec597baeb04f7e1b`
-        await Task.CompletedTask;
-    }
+    [TestCase(
+        "0xa66ec67b117f57388da53271f00c22a68e6c297b564f67c5904e6f2662881875",
+        "0xe168b70ac8a6f7d90734010030801fbb2dcce03a657155c4024b36ba8d1e3926"
+        )]
+    [Parallelizable(ParallelScope.None)]
+    public override Task forkchoiceUpdatedV1_should_communicate_with_boost_relay_through_http(string blockHash, string parentHash)
+        => base.forkchoiceUpdatedV1_should_communicate_with_boost_relay_through_http(blockHash, parentHash);
 
     class MergeAuRaTestBlockchain : MergeTestBlockchain
```

Comment:
`MergeAuRaTestBlockchain` resolves internally the Ethereum mainnet withdrawal processor (https://github.com/NethermindEth/nethermind/blob/master/src/Nethermind/Nethermind.Consensus/Withdrawals/WithdrawalProcessor.cs) instead of Aura version of withdrawal processor. It will be easily notified by debugging tests, for example Nethermind.Merge.AuRa.Test.newPayloadV2_should_validate_withdrawals

Reply:
This point is even more important because of the current state of hive tests for withdrawals.